addons/mail/static/src/core/common/chat_bubble.xml

```diff
@@ -28,7 +28,7 @@
                 <t t-call="mail.message_preview_prefix">
```

Comment:
Shouldn't the content of `message_preview_prefix` also take email into account?
Why do we have a js and xml version of this code?